tests/lib.rs

```diff
@@ -67,3 +67,18 @@ fn check_write() {
   assert_eq!(se.len(), 8u64);
   assert_eq!(se.get_storage().get(name),vec![0x11]);
 }
+
+#[test]
+
+fn check_read() {
+  let name = vec![0x11];
+  let mut my_storage = MyStorage{name: vec![0x11]};
+  let mut se = SelfEncryptor::new(&mut my_storage as &mut Storage, datamap::DataMap::None);
```

Comment:
So next test is to create a new encryptor and pass teh correct datamap to the new() call, when the close() if completed of course.